core/src/preflight.rs

```diff
@@ -257,14 +262,33 @@ async fn prepare_taiko_chain_input(
             RaikoError::Preflight("Beacon RPC URL is required for Taiko chains".to_owned())
         })?;
         let blob = get_blob_data(&beacon_rpc_url, slot_id, blob_hash).await?;
-        let commitment = eip4844::calc_kzg_proof_commitment(&blob).map_err(|e| anyhow!(e))?;
+        let commitment = calc_kzg_proof_commitment(&blob).map_err(|e| anyhow!(e))?;
+        let blob_proof = match blob_proof_type {
+            BlobProofType::KzgVersionedHash => None,
+            BlobProofType::ProofOfEquivalence => {
+                let (x, y) = proof_of_equivalence(&blob, &commitment_to_version_hash(&commitment))
+                    .map_err(|e| anyhow!(e))?;
+
+                println!("x {:?} y {:?}", x, y);
+                println!(
+                    "calc_kzg_proof_with_point {:?}",
+                    calc_kzg_proof_with_point(&blob, ZFr::from_bytes(&x).unwrap()).unwrap()
+                );
```

Comment:
probably want to change this to info/debug

Reply:
second it!